### PR TITLE
006 view role responsibilities

### PR DESCRIPTION
**Pull request by Jadevious**
Ticket: https://trello.com/c/yKomfD2p

### CODE REVIEW

File: src/main/java/com/kainos/ea/dao/JobsDao.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 public class JobsDao {
     public List<Job> getJobs(Connection c) throws SQLException, DatabaseConnectionException {
         Statement st = c.createStatement();
-        PreparedStatement statement = c.prepareStatement("SELECT id, name, job_description FROM job_roles");
+        PreparedStatement statement = c.prepareStatement("SELECT id, name, job_description, job_responsibilities FROM job_roles");
```
Comment (boydstepnew) [inline src/main/java/com/kainos/ea/dao/JobsDao.java]: Perhaps the prefix of "job_" in the job_roles table is unnecessary? How about changing it to description and responsibilities.

File: src/main/java/com/kainos/ea/models/Job.java [hunk 2]
```diff
@@ -17,14 +17,23 @@ public class Job {
...
+    }
 
     public Job(int id, String name, String job_description) {
         this.id = id;
         this.name = name;
         this.job_description = job_description;
     }
 
-    public Job() {
+    public Job(int id, String name, String job_description, String job_responsibilities) {
```
Comment (boydstepnew) [inline src/main/java/com/kainos/ea/models/Job.java]: Can drop the prefix here too, we're in a job class. `Job.responsibilities` reads nicer than `Job.job_responsibilities`

Comment (Jadevious) [reply src/main/java/com/kainos/ea/models/Job.java]: Sure thing, we've made the changes for all active feature branches due to the global nature of changes to the SQL table structure.

Comment (boydstepnew) [reply src/main/java/com/kainos/ea/models/Job.java]: Okay, although that's what merging is for - so changes don't need duplicated across feature branches. :)

**Review by boydstepnew — APPROVED** [review]

**Review by conormcgee480 — APPROVED** [review]

**Pull request merged**